internal/xds/translator/testdata/in/xds-ir/accesslog-cel.yaml

```diff
@@ -51,6 +51,3 @@ http:
       - endpoints:
         - host: "1.2.3.4"
           port: 50000
-    directResponse:
```

Comment:
why this is removed?

Reply:
it wasn't adding any value to the access log tests

Reply:
there's no new test?

Reply:
maybe worth to do in an seperate PR, and add some test for DirectReponse in this one.

Reply:
if I keep this, need there will be new out files generated, because the `body` is now translated to xds

Reply:
ok